packages/rspack-test-tools/tests/configCases/resolve/axios-0.27.2/rspack.config.js

```diff
@@ -1,4 +1,6 @@
 /** @type {import("@rspack/core").Configuration} */
 module.exports = {
-	target: "web"
+	resolve: {
+		aliasFields: ["browser"]
```

Comment:
Normal cases has this aliasFields and no target by default, this file is not used before